test/NuGet.Core.Tests/NuGet.Protocol.Tests/HttpSource/HttpSourceAuthenticationHandlerTests.cs

```diff
@@ -492,7 +492,7 @@ public static IEnumerable<object[]> GetHttpContent()
             yield return new object[] { multipartFormDataContent };
         }
 
-        [Theory]
+        [Theory(Skip = "https://github.com/NuGet/Home/issues/9685")]
```

Comment:
This test is not causing hangs on Linux or Windows. I suggest using PlatformFactAttribute instead.